incubator/kitura/image/config/app-deploy.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: appsody.dev/v1beta1
+kind: AppsodyApplication
+metadata:
+  name: APPSODY_PROJECT_NAME
+spec:
+  # Add fields here
+  version: 1.0.0
+  applicationImage: APPSODY_DOCKER_IMAGE 
+  stack: APPSODY_STACK
+  service:
+    type: NodePort
+    port: APPSODY_PORT
```

Comment:
Seeing as this stack provides health/metrics endpoints, you could add liveness/readiness probes to this service